src/menusys.c

```diff
@@ -736,13 +740,8 @@ void menuRenderMenu()
         // render, advance
         fntRenderString(gTheme->fonts[0], 320, y, ALIGN_CENTER, 0, 0, submenuItemGetText(&it->item), (cp == sitem) ? gTheme->selTextColor : gTheme->textColor);
         y += spacing;
-        if (gHDDStartMode && gEnableWrite) {
-            if (cp == 7)
-                y += spacing / 2;
-        } else {
-            if (cp == 6)
-                y += spacing / 2;
-        }
+        if (cp == (MENU_ABOUT - 1))
+            y += spacing / 2;
```

Comment:
If you plan to make changes not related to OSD you need to clean up your commit history - so this will be separate commit.

Reply:
if we will use squash and merge, these changes will be merged in osd commit. Currently, your commit history is a mess, so either move non-related to OSD changes into separate pr, or clean your commit history

Reply:
@israpps 
“git rebase -i origin your-branch-name” Will open a text window where you can squash commits, rename, drop, fix up etc

“git push -f” To force push the changes after rebase

Reply:
> If you plan to make changes not related to OSD you need to clean up your commit history - so this will be separate commit.

Do I really have to split it? I had to change that code anyways...

Anyway, I'll see what I do in a few hours, when I'm in front of the PC